### DISCUSSION TITLE
제페토 1.1.5에서 혹시 Timeline 지원하나요?

### DISCUSSION THREAD

**Discussion by hyunjoon-park-metaspace** [naverz/zepeto-studio-kor#654] [5. Q&A]
재생해보려는데 안 되서요, 혹시 제페토가 공식적으로 Timeline을 지원하고 있는지 궁금합니다 빠른 답변 바랍니다 감사합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. Timeline은 빠른 시일 내 지원 예정이며 업데이트 시 안내 드리겠습니다.
감사합니다.
